test/gurka/test_matrix.cc

```diff
@@ -92,7 +92,7 @@ class MatrixTrafficTest : public ::testing::Test {
                             {{"service_limits.max_timedep_distance_matrix", "50000"},
                              {"mjolnir.traffic_extract",
                               "test/data/matrix_traffic_allowed/traffic.tar"},
-                             {"mjolnir.timezone", "test/data/tz.sqlite"},
+                             {"mjolnir.timezone", VALHALLA_BUILD_DIR "test/data/tz.sqlite"},
```

Comment:
this was the reason they were failing on my machine

Reply:
yeah i had a branch (long stale) that fixed all of these but there were other thigns on it nils suggested that i never got around to